### DISCUSSION TITLE
Is there a way to dump all executed instructions?

### DISCUSSION THREAD

**Discussion by Theldus** [86Box/86Box#3653] [Q&A] [answered]
I'm running an old piece of software on 86Box, and I'd like to be able to trace the entire execution flow of this code. Is there a way to dump all the instructions executed up to this point?

I know that in QEMU, this can be achieved through plugins, but I'm not sure if there's something similar available in 86Box.

**Comment by jriwanek** [comment, marked as answer]
86Box doesn't currently have this feature

**Reply by Theldus** [reply]
Thanks, I managed to do this using GDB to debug. If I remember correctly, I also managed to do something similar by modifying the source code, but I don't remember exactly what I did, but thanks anyway.
